website/docs/metrics.md

```diff
@@ -201,3 +201,23 @@ These `HoodieMetrics` can then be plotted on a standard tool like grafana. Below
 <figure>
     <img className="docimage" src={require("/assets/images/hudi_commit_duration.png").default} alt="hudi_commit_duration.png"  />
 </figure>
+
+## List of metrics:
+
+1. totalPartitionsWritten
```

Comment:
Even though the names are self-explanatory, I think it would be good add a line about each metric and possibly point to the class where these metrics are captured. Wdyt?

Reply:
Sure. Will do.

Reply:
Done @codope. Please review and let me know of your thoughts.